ansible/main.yml

```
@@ -0,0 +1,110 @@
+---
+- hosts: all
+  vars:
+    app_path: /home/ubuntu/RD-WATCH
```

```suggestion
    app_path: "{{ ansible_facts['user_dir'] }}/RD-WATCH"
```
This avoids hardcoding the home dir.